pkg/process/procutil/process_linux.go

```diff
@@ -408,3 +518,37 @@ func trimAndSplitBytes(bs []byte) []string {
 
 	return components
 }
+
+// bootTime returns the system boot time expressed in seconds since the epoch.
+// the value is extracted from "/proc/stat"
+func bootTime(hostProc string) (uint64, error) {
+	filePath := filepath.Join(hostProc, "stat")
+	content, err := ioutil.ReadFile(filePath)
+	if err != nil {
+		log.Debugf("Unable to read stat file from %s: %s", filePath, err)
+		return 0, nil
+	}
+
+	index := 0
```

Comment:
What do you think about calling this `lineStart`?

Reply:
Updated.